module/zfs/arc.c

```diff
@@ -1947,7 +1948,7 @@ arc_fill_hdr_crypt(arc_buf_hdr_t *hdr, kmutex_t *hash_lock, spa_t *spa,
  * arc_buf_fill().
  */
 static void
-arc_buf_untransform_in_place(arc_buf_t *buf, kmutex_t *hash_lock)
+arc_buf_untransform_in_place(arc_buf_t *buf)
```

Comment:
Just for context the `hash_lock` still actually may be checked here through the `HDR_EMPTY_OR_LOCKED()` check.  It doesn't need so this looks fine.